tests/smoke/events.js

```diff
@@ -6,7 +6,10 @@
 (function () {
 	bender.editor = {
 		config: {
-			enterMode: CKEDITOR.ENTER_P
+			enterMode: CKEDITOR.ENTER_P,
+			nanospell: {
+				autostart: false
+			}
```

Comment:
Seems like we might want to pull this up a level into a helper that all of our tests use. My guess is that autostart will get us into trouble pretty much everywhere.